bedrock/products/redirects.py

```diff
@@ -2,7 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-from bedrock.redirects.util import redirect
+from bedrock.redirects.util import mobile_app_redirector, redirect
+
+
+def mobile_app(request, *args, **kwargs):
+    qs = request.META.get("QUERY_STRING", "")
+    campaign = None
+    product = "vpn"
+
+    product_options = ["vpn"]
+
+    campaign_options = [
+        "vpn-landing-page",
+        "vpn-pricing-page",
+    ]
+
+    for p in product_options:
+        if f"product={p}" in qs:
+            product = p
+
+    for c in campaign_options:
+        if f"campaign={c}" in qs:
+            campaign = c
```

Comment:
For both of these, it might be worth adding a `break` after assigning to `campaign` or `product` unless we're 100% sure that there won't be multiple querystring matches (eg someone adds both `?product=vpn-landing-page&product=vpn_pricing_page` but this is pretty unlikely

Reply:
Hmm, actually I'm not 1005 sure if we do want that, since if we had both `firefox-whatsnew` and `firefox-whatsnew-131` in the campaign list, then that might match the former and return early, before then correctly matching the latter. Does that make sense?

Reply:
Actually I wonder if we need to do a more strict comparison here, rather than `if f"campaign={c}" in qs:` ?

Reply:
> ...then that might match the former and return early, before then correctly matching the latter. Does that make sense?

I guess it depends on what the desired behaviour is. Right now if we have `?campaign=foo&campaign=bar`, we'd get `bar` not `foo`, so either way one thing gets lost, so ideally we'd never have a repeated querystring key